README.md

```diff
@@ -1,2 +1,25 @@
-# who-we-are
-This repository presents this organization
+# Friends of PrestaShop
+
+This repository presents Friends of PrestaShop community, its missions, intents and publics.
+
+## Publics
+
+We are PrestaShop integrators and developers, working together on modules themes and additional resources.
+
+> Is this organization and its differents channels open for merchants and complete beginners?
+
+No, please refers to the official forum of PrestaShop if you're a merchant or to the official User or Developers documentations if you needs to learn PrestaShop basics.
+
+## Intents
```

Comment:
Maybe `Focus`? Not sure though.

Reply:
maybe something easier for everyone: "What we do"

wdyt ?

Reply:
:ok_hand:

Reply:
ok